imsv-docs-docusaurus/openapi/endpoints/funding-sources/create-funding-source.yaml

```diff
@@ -28,22 +27,17 @@ post:
 
         **FORBIDDEN**
         Insufficient permissions to perform the action.
-
-        **SIGNATURE_DENIED**
-        The signature was rejected by the ERC-1271 contract `isValidSignature` function.
-
-        **SIGNATURE_INVALID**
-        The message signature was invalid.
-
-        **ADDRESS_MISMATCH**
-        The `ownerAddress` does not match the EIP-4361 message address.
       content:
         application/json:
           schema:
             $ref: "../../models/api-error-403.yaml"
 
     "400":
-      description: Request fields are invalid
+      description: |
```

Comment:
400 is the same on every endpoint. Should either not document or refer to common definition.